src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -565,12 +565,9 @@ public class OpenFileFromJsonCommand : RecordableCommand
             /// </summary>
             /// <param name="fileContents">The Json content of a file.</param>
             /// <param name="forceManualExecutionMode">Should the file be opened in manual execution mode?</param>
-            /// <param name="forceAutomaticWithoutRun">This parameter, if true, will prohibit the graph from executing when RunMode is set to Automatic</param>
-            public OpenFileFromJsonCommand(string fileContents, bool forceManualExecutionMode = false, bool forceAutomaticWithoutRun = false)
+            public OpenFileFromJsonCommand(string fileContents, bool forceManualExecutionMode = false)
             {
                 FileContents = fileContents;
-                ForceManualExecutionMode = forceManualExecutionMode;
```

Comment:
Don't you have to keep this line?

Reply:
@aparajit-pratap was deleted by mistake, it was re-added in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12947/commits/9e9530081f1fe1a44784ee009ee220886e4dc7c3